src/translations.js

```diff
@@ -158,6 +163,7 @@ const statCardLocales = ({ name, apostrophe }) => {
       sk: "Účasti",
       tr: "Katkı Verildi",
       pl: "Udziały",
+      vi: "Đã Đóng Góp"
```

Comment:
@nobiit Thanks for your translation. I don't speak Vietnamese, but I'm reviewing your pull request using google translate. I was just wondering why you're using this over `Góp phần`, which google translate suggests is?

Reply:
`Contribute` means `Đóng góp`, and I use `Đã Đóng Góp` to refer to past contribute. `Góp phần`, when translate into English, is `Contribute`, but this word is not commonly used to refer to contribute in my country